composer.json

```diff
@@ -12,14 +12,15 @@
     ],
     "require": {
         "php": ">=7.1.0",
```

Comment:
Since your using `mixed` return types you can just bump this to `^8.0` since this want work on anything less

Reply:
++